addons/pos_gift_card/static/src/js/GiftCardPopup.js

```diff
@@ -4,7 +4,7 @@ odoo.define("pos_gift_card.GiftCardPopup", function (require) {
   const AbstractAwaitablePopup = require("point_of_sale.AbstractAwaitablePopup");
   const Registries = require("point_of_sale.Registries");
 
-  const { useState } = owl;
+  const { useState, onPatched, useComponent} = owl.hooks;
```

Comment:
Native hooks are now at `owl`'s top level.
```suggestion
  const { useState, onPatched, useComponent} = owl;
```